internal/app/machined/pkg/system/services/etcd.go

```diff
@@ -341,13 +356,14 @@ func (e *Etcd) argsForInit(ctx context.Context, r runtime.Runtime) error {
 		"data-dir":              constants.EtcdDataPath,
 		"listen-peer-urls":      "https://" + net.FormatAddress(listenAddress) + ":2380",
 		"listen-client-urls":    "https://" + net.FormatAddress(listenAddress) + ":2379",
-		"cert-file":             constants.KubernetesEtcdPeerCert,
-		"key-file":              constants.KubernetesEtcdPeerKey,
+		"client-cert-auth":      "true",
+		"cert-file":             constants.KubernetesEtcdClientCert,
```

Comment:
this feels wrong: we should use server cert here (not client cert, and not peer cert)

Reply:
May be I need to rename it.
EtcdClientCert - this is a server certificate for client connections.

Reply:
looks like we have a mixed client/server cert there, that's why I asked

Reply:
renamed EtcdClientCert -> EtcdCert